src/Orchard.Cms.Web/Modules/Orchard.Lists/Views/ListPart.Feed.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@using Orchard.Feeds.Models
+@using Orchard.ResourceManagement
+@using Orchard.ContentManagement
+
+@inject IResourceManager ResourceManager
+@inject IContentManager ContentManager
+
+@{ 
+    ContentItem contentItem = Model.ContentItem;
+    var contentItemMetadata = ContentManager.PopulateAspect<ContentItemMetadata>(contentItem);
+    var feedMetadata = ContentManager.PopulateAspect<FeedMetadata>(contentItem);
+    ResourceManager.RegisterLink(new LinkEntry
+    {
+        Rel = "alternate",
+        Type = "application/rss+xml",
```

Comment:
It is encoded when displayed in the source of the page:
application/rss&#x2B;xml